plugins/nf-co2footprint/src/resources/CO2FootprintReportTemplate.html

```diff
@@ -146,47 +177,54 @@ <h4>Workflow execution completed unsuccessfully!</h4>
+          <span class="metric-title">CO<sub>2</sub>e emissions</span>
+        </div></div>
+        <div class="col-12 col-md-6 col-xl"><div class="metric-card card my-3">
+          <span class="metric">${co2_totals.energy}  Wh</span>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><mask id="ipSEnergySocket0"><g fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="4"><path d="M24 44c11.046 0 20-8.954 20-20S35.046 4 24 4S4 12.954 4 24s8.954 20 20 20Zm4-23v-5m-8 5v-5"/><path fill="#fff" d="M24 32a8 8 0 0 0 8-8v-3H16v3a8 8 0 0 0 8 8Z"/><path d="M24 44V32"/></g></mask><path fill="currentColor" d="M0 0h48v48H0z" mask="url(#ipSEnergySocket0)"/></svg>
+          <span class="metric-title">Energy consumption</span>
+        </div></div>
+        <div class="col-12 col-md-6 col-xl"><div class="metric-card card my-3">
+          <span class="metric">${co2_totals.tree}</span>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><path fill="currentColor" d="M378.31 378.49L298.42 288h30.63c9.01 0 16.98-5 20.78-13.06c3.8-8.04 2.55-17.26-3.28-24.05L268.42 160h28.89c9.1 0 17.3-5.35 20.86-13.61c3.52-8.13 1.86-17.59-4.24-24.08L203.66 4.83c-6.03-6.45-17.28-6.45-23.32 0L70.06 122.31c-6.1 6.49-7.75 15.95-4.24 24.08C69.38 154.65 77.59 160 86.69 160h28.89l-78.14 90.91c-5.81 6.78-7.06 15.99-3.27 24.04C37.97 283 45.93 288 54.95 288h30.63L5.69 378.49c-6 6.79-7.36 16.09-3.56 24.26c3.75 8.05 12 13.25 21.01 13.25H160v24.45l-30.29 48.4c-5.32 10.64 2.42 23.16 14.31 23.16h95.96c11.89 0 19.63-12.52 14.31-23.16L224 440.45V416h136.86c9.01 0 17.26-5.2 21.01-13.25c3.8-8.17 2.44-17.47-3.56-24.26z"/></svg>
+          <span class="metric-title">Tree months</span>
+        </div></div>
+        <div class="col-12 col-md-6 col-xl"><div class="metric-card card my-3">
+          <span class="metric">${co2_totals.car}</span>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="currentColor" d="m5 11l1.5-4.5h11L19 11m-1.5 5a1.5 1.5 0 0 1-1.5-1.5a1.5 1.5 0 0 1 1.5-1.5a1.5 1.5 0 0 1 1.5 1.5a1.5 1.5 0 0 1-1.5 1.5m-11 0A1.5 1.5 0 0 1 5 14.5A1.5 1.5 0 0 1 6.5 13A1.5 1.5 0 0 1 8 14.5A1.5 1.5 0 0 1 6.5 16M18.92 6c-.2-.58-.76-1-1.42-1h-11c-.66 0-1.22.42-1.42 1L3 12v8a1 1 0 0 0 1 1h1a1 1 0 0 0 1-1v-1h12v1a1 1 0 0 0 1 1h1a1 1 0 0 0 1-1v-8l-2.08-6Z"/></svg>
+          <span class="metric-title">Km by car</span>
```

Comment:
As it's a SI unit, should not be capitalised I think.
```suggestion
          <span class="metric-title">km by car</span>
```